internal/checkers/negative_postive.go

```diff
@@ -25,6 +25,8 @@ import (
 //	assert.False(t, a <= 0)
 //	assert.False(t, 0 >= a)
 //
+// Typed zeros (like `int(0)`, `int(8)`, … `uint(64)`) are also supported.
```

Comment:
invalid place of comment – you broke the sentence "detects ... and requires ..."